packs/data/bestiary-effects.db/effect-ectoplasmic-form-physical.json

```diff
@@ -0,0 +1,76 @@
+{
+    "_id": "QMcmhifNQOPguv2t",
+    "img": "systems/pf2e/icons/spells/dinosaur-form.webp",
+    "name": "Effect: Ectoplasmic Form (Physical)",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>The creature looses the incorporeal trait, the immunity to precision damage and all resistances. It gains temporary hit points and an increased armor class arroding to the specific creature, its jaws strikes deal piercing and its claws strikes deal slashing damage instead of negative.</p>"
```

Comment:
```suggestion
            "value": "<p>The creature loses the incorporeal trait, the immunity to precision damage and all resistances. It gains temporary hit points and an increased armor class accoding to the specific creature, its jaws strikes deal piercing and its claws strikes deal slashing damage instead of negative.</p>"
```